### DISCUSSION TITLE
Tesla android Wi-Fi

### DISCUSSION THREAD

**Discussion by WillE726** [tesla-android/issue-tracker#254] [Q&A]
Wi-Fi is disconnecting and then can't get the Wi-Fi to come back without restarting the raspberry pi?  Any suggestions?  Thanks

**Comment by mikegapinski** [comment]
It can be disabled only if the system terminated, this happens when there is some under voltage present. Check the power source and the cable

**Comment by hal9** [comment]
It happened to me. A bad cable can work intermittently.  usbc cable has a chip and modulate his voltage.
